db/migrate/20190308134512_create_async_callbacks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAsyncCallbacks < ActiveRecord::Migration
+  def change
+    create_table :async_callbacks do |t|
+      t.string :uuid, :null => false
+      t.integer :timestamp, :null => false
+      t.string :listener_uri
```

Comment:
when we get into more interesting transports, just documenting we'll be able to include other cool stuff like 'icmp://blah' and 'dns://foo' in here as well.